homeassistant/components/vallox/translations/sv.json

```diff
@@ -0,0 +1,18 @@
+{
+    "config": {
+        "error": {
+            "already_configured": "Enheten är redan tillagd",
+            "cannot_connect": "Det gick inte att ansluta till Vallox",
+            "invalid_host": "Ogiltigt värdnamn eller IP-adress",
+            "unknown": "Okänt fel"
+        },
+        "step": {
+            "user": {
+                "data": {
+                    "host": "Värdnamn eller IP-adress",
+                    "name": "Namn"
+                }
+            }
+        }
+    }
+}
```

Comment:
This will be overwritten by the translations system.  Please remove any other language than english and add it to lokalise later